bootstrap-salt.sh

```diff
@@ -996,7 +996,8 @@ __gather_linux_system_info() {
             # Skip setting DISTRO_NAME this time, splitting CamelCase has failed.
             # See https://github.com/saltstack/salt-bootstrap/issues/918
             [ "$n" = "$DISTRO_NAME" ] && DISTRO_NAME="" || DISTRO_NAME="$n"
-        elif [ "${DISTRO_NAME}" = "openSUSE project" ]; then
+        elif [ "$( echo "${DISTRO_NAME}" | grep openSUSE )" != "" ]; then
```

Comment:
this could be a bit prettier with something like
```bash
elif echo "${DISTRO_NAME}"|grep -q openSUSE; then
```
in my opinion

Reply:
I agree your suggestion is cleaner.
In this PR the conditional (line 999) now follows the same style as most other conditionals. 
https://github.com/saltstack/salt-bootstrap/blob/fbe3759e4f750aa59c0edb628b4dcb3e74f836c3/bootstrap-salt.sh#L993
The dilemma is that I should, in the first instance, follow same format as code around you.  So best to keep your suggestion for separate PR.

Reply:
I agree, I didn't realize how much much that style was used around it until after I commented.

Reply:
Yes, in my original update I done something else but then noticed the style.